packages/blade/src/components/Tooltip/_decisions/decisions.md

```diff
@@ -0,0 +1,107 @@
+  - [Examples:](#examples)
+    - [Basic](#basic)
+- [Library](#library)
+- [Motion](#motion)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=40540-559304&t=tmTrf3xJU6oj59fM-0) to all variants of the Tooltip component
+
+## `Tooltip` API
+
+| Prop          | Type                                                                     | Default     | Description                                                               | Required |
+| ------------- | ------------------------------------------------------------------------ | ----------- | ------------------------------------------------------------------------- | -------- |
+| content       | `string`                                                                 | `undefined` | Content of the tooltip                                                    | ✅       |
+| placement     | `top, top-start, top-end, left, right, bottom, bottom-start, bottom-end` | `top`       | Placement of tooltip                                                      | ✅       |
+| children      | `React.ReactNode`                                                        | `undefined` | Trigger component for tooltip, Accepts any interactive element            | ✅       |
+| isOpen        | `boolean`                                                                | `undefined` | Controls the tooltip state                                                |          |
+| defaultIsOpen | `boolean`                                                                | `undefined` | If true, the tooltip will be visible initially                            |          |
+| onDismiss     | `Callback`                                                               | `undefined` | Called when tooltip is closed, either by mouseout or users pressing `Esc` |          |
+
+### Examples:
+
+#### Usage
+
+```js
+// basic, uncontrolled
+<Tooltip content="Amount reversed to customer bank account" placement="top">
+  <IconButton icon={InfoIcon} accessibilityLabel="Refund" />
+</Tooltip>;
+
+// controlled
+const [isOpen, setIsOpen] = React.useState(false);
+
+<Tooltip
+  onDismiss={() => setIsOpen(false)}
+  isOpen={isOpen}
+  content="Amount reversed to customer bank account"
+>
+  <IconButton icon={InfoIcon} accessibilityLabel="Refund" />
+</Tooltip>;
+
+// default open
+<Tooltip defaultIsOpen={true} content="Amount reversed to customer bank account">
+  <IconButton icon={InfoIcon} accessibilityLabel="Refund" />
+</Tooltip>;
+```
+
+## Library
+
+We will be using [FloatingUI](https://floating-ui.com/) to position the tooltip & handle the basic tooltip logic. Floating UI supports both Web & ReactNative.
+
+## Implementation detail nuances:
+
+To make FloatingUI work with Blade components seamlessly there are few things we need to modify:
+
+1. Expose interaction props in all the trigger components
```

Comment:
We had some usecases for analytics things and doing navbar like popovers on website. So this might anyway be required.

Reply:
Yeah we could add these. Tho note that we are exposing onMouseMove (which is required for floatingUI) not onMouseOver, mouseMove should not be used for analytics.